portal-frontend/src/app/services/parcel/parcel.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../environments/environment';
+import { ToastService } from '../toast/toast.service';
+import { ParcelLookupDto } from './parcel.dto';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ParcelService {
+  private serviceUrl = `${environment.apiUrl}/parcel`;
+
+  constructor(private httpClient: HttpClient, private toastService: ToastService) {}
+
+  async getByPidPin(pidPin: string) {
```

Comment:
Unit test for this one? Or are we ignoring Angular for now, like previously with ALCS?

Reply:
Unit test added